src/quo/components/wallet/token_input/view.cljs

```diff
@@ -107,7 +108,8 @@
                      :selection-color          customization-color
                      :show-soft-input-on-focus show-keyboard?
                      :on-selection-change      handle-selection-change
-                     :selection                (clj->js selection)}
+                     :selection                (clj->js selection)
```

Comment:
Not related to this PR, but I'm wondering why do we do `clj->js` Isn't it something that reagent already does?